opm/grid/GraphOfGrid.hpp

```diff
@@ -162,9 +162,24 @@ class GraphOfGrid{
         return wells;
     }
 
+    /// \brief Contract a layer of verices around each wells into it
+    ///
+    /// Representing a well by one node guarantees that the well won't
+    /// be split over several processes. Giving the well an extra layer
+    /// of cells distances that well from the subdomain boundary.
+    void addWellBuffer ();
+    void addWellBuffer (int layers)
+    {
+        for (int i=0; i<layers; ++i)
+        {
+            addWellBuffer();
+        }
+    }
+
 private:
     /// \brief Create a graph representation of the grid
-    void createGraph (); // edge weight=1
+    /// If transmissibilities are not supplied, edge weight=1
```

Comment:
`\brief` went missing in action. Without that there will be no short description in doxygen. Is this intentional.

Reply:
`\brief` did not get deleted. I added a second line to the function description.

Reply:
then the comment is now less brief as all lines are used until the first empty.

Reply:
I will add an empty line there.